src/components/Launch.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Image } from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet'
+
+const Launch = () => {
+  return (
+    <Image source={require('../../images/fulllaunch.png')} style={styles.container} />
+  )
+}
+
+const styles = EStyleSheet.create({
+  container: {
+    flex: 1,
+    width: null,
+    height: null,
```

Comment:
Yeah it's an RN thing :-( https://github.com/facebook/react-native/issues/950 Basically it resets any sizing and forces RN to calculate the image size based on device size (alternatively I could pull in the device size and manually calculate the size of the image myself).  Otherwise trying to position: absolute doesn't work, neither does just having flex: 1 and omitting the width/height.